src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -843,17 +850,21 @@ protected function checkCategories($hookData, Github $github, Logger $logger, $p
 		// Get the files tha gets changed with this Pull Request
 		$files = $this->getChangedFilesByPullRequest($hookData, $github, $logger, $project);
 
-		// The new categories based on the current code of the PR
-		$newCategories = $this->checkFilesAndAssignCategory($files);
+		// If there are no changed files do nothing here.
+		if (!empty($files))
```

Comment:
I would make here the same way.

if (empty($files))
{
        // If there are no changed files return
        return;
}